sktime/contrib/critical_difference.py

```diff
@@ -0,0 +1,600 @@
+# -*- coding: utf-8 -*-
+"""Standalone function to plot critical difference diagram."""
+
+__author__ = ["Svea Meyer"]
+
+import numpy as np
+from scipy.stats import rankdata, find_repeats, distributions
+import math
+from sktime.utils.validation._dependencies import _check_soft_dependencies
+
+_check_soft_dependencies("matplotlib")
+import matplotlib.pyplot as plt
+
+
+def _check_friedman(n_strategies, n_datasets, ranked_data, alpha):
+    """
+    Check whether Friedman test is significant.
+
+    Larger parts of code copied from scipy.
```

Comment:
if it's copied - why can't you use `scipy` directly, leaving the logic in `scipy`?